app/src/main/java/io/github/drp08/studypal/presentation/screens/AddSubjectScreen.kt

```diff
@@ -101,6 +109,26 @@ data object AddSubjectScreen : Screen {
                         confidence = state.confidenceLevel,
                         onConfidenceChange = { viewModel.on(ChangeConfidence(it)) }
                     )
+                    var i = 0
+                    var topics : MutableList<String> = mutableListOf() //Todo: Add to view model
```

Comment:
Would you wanna add the topics to viewModel?

Reply:
no

Reply:
view model not in this branch yet

Reply:
needs to wait til merged with main

Reply:
Why deal with it separately when we can get it done with it right away?

Reply:
cause I need the rest of the framework, and merge conflicts are then gonna be a nightmare for that file

Reply:
Okay cool